examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { CommandBus } from '@nestjs/cqrs';
+import { CreateBpiSubjectCommand } from '../capabilities/createBpiSubject/createBpiSubject.command';
+import { BpiSubject } from '../models/bpiSubject';
+import { CreateBpiSubjectDto } from './dtos/request/createBpiSubject.dto';
+
+@Controller("subjects")
+export class SubjectController {
+  constructor(private commandBus: CommandBus) {}
+
+  // TODO: DTO validation
```

Comment:
what we will validate on this level and what on agent level?

Reply:
On this level we validate the input - request DTO. For example foo must be string and bar must not be emtpy. 
On the agent level we validate business logic. For example, entity X cannot be edited if in status Y.